test/support/still/case.ex

```diff
@@ -9,6 +9,7 @@ defmodule Still.Case do
         Application.put_env(:still, :pass_through_copy, [])
 
         Still.Utils.rm_output_dir()
+        Still.Utils.mk_output_dir()
```

Comment:
Why do we need this?

Reply:
For the output folder to exist

Reply:
Should we not remove it in the line above, then? Just remove the contents

Reply:
No, because there are files we want to remove there.

Reply:
Yeah, but can't we remove the contents in a single system call? `Still.Utils.clean_output_dir/0` that does `File.rm(dir <> "/*")`

Reply:
I'm not sure if having one system call instead of those two is better, but sure 👍